File: OLSource1/v1/VS_officedev/codesnippet/CSharp/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_1.cs
            if (Globals.ThisAddIn.Application.Documents.Count > 0)
            {
                Microsoft.Office.Interop.Word.Document nativeDocument =
                    Globals.ThisAddIn.Application.ActiveDocument;
                Microsoft.Office.Tools.Word.Document vstoDocument =
                    Globals.Factory.GetVstoObject(nativeDocument);
            }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_1.vb
        If Globals.ThisAddIn.Application.Documents.Count > 0 Then
            Dim NativeDocument As Microsoft.Office.Interop.Word.Document = _
                Globals.ThisAddIn.Application.ActiveDocument
            Dim VstoDocument As Microsoft.Office.Tools.Word.Document = _
                Globals.Factory.GetVstoObject(NativeDocument)
        End If